mmseg/version.py

```diff
@@ -0,0 +1,16 @@
+__version__ = '0.5.0'
+short_version = __version__
```

Comment:
`short_version` can be removed. In mmdet it is only kept for backward compatibility.